src/utils/primitives.js

```diff
@@ -8,4 +8,8 @@ function isNum(candidate) {
   return !isNaN(parseInt(candidate));
 }
 
-export { encodeGetParams, isNum };
+function emptyObject(obj) {
```

Comment:
personally I'd call this `isObjectEmpty`. consistent with `isNum` above and functions like `isNaN`